install-dependencies_lducas.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#git clone -b fpylll-changes https://github.com/lducas/fplll
+cd fplll
+./autogen.sh
+if test "$1" != ""; then
```

Comment:
pull requests cover branches, not a set of commits, so anything you commit after will be included. The best strategy is to create a branch off your latest commit you want to include and then send a new pull request from that branch.